suite-native/module-settings/src/components/CoinsSettings.tsx

```diff
@@ -1,66 +0,0 @@
-import { Box, Chip } from '@suite-native/atoms';
-import { CryptoIcon, CryptoIconName } from '@suite-common/icons';
-
-import { SettingsSection } from './SettingsSection';
-
-const dummyCoins: { title: string; iconName: CryptoIconName; description?: string }[] = [
-    {
-        title: 'Bitcoin',
-        iconName: 'btc',
-    },
-    {
-        title: 'Litecoin',
-        iconName: 'ltc',
-    },
-    {
-        title: 'Ethereum',
-        iconName: 'eth',
-        description: 'inc. Tokens',
-    },
-    {
-        title: 'Ethereum Classic',
-        iconName: 'eth',
-    },
-    {
-        title: 'XRP',
-        iconName: 'ltc',
-    },
-    {
-        title: 'Dash',
-        iconName: 'ltc',
-    },
-    {
-        title: 'Bitcoin Cash',
-        iconName: 'ltc',
-    },
-    {
-        title: 'Bitcoin Gold',
-        iconName: 'ltc',
-    },
-    {
-        title: 'DigiByte',
-        iconName: 'ltc',
-    },
-];
-
-export const CoinsSettings = () => (
-    <SettingsSection
-        title="Coins"
-        subtitle="2 coins active, 1000 tokens active"
-        rightIconName="settings"
-        onRightIconPress={() => console.warn('coin select')}
-    >
-        <Box flexDirection="row" flexWrap="wrap" alignItems="center">
-            {dummyCoins.map(item => (
-                <Box marginRight="small" marginBottom="small" key={item.title}>
-                    <Chip
```

Comment:
Is Chip used anywhere?

Reply:
Good catch, no it's not. Removed https://github.com/trezor/trezor-suite/pull/10698/commits/514c3fff39663fdb40d5a5c090130d8de4d9918b